src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -2132,10 +2132,6 @@ private function registerJsonStreamerConfiguration(array $config, ContainerBuild
         $container->setParameter('.json_streamer.stream_writers_dir', '%kernel.cache_dir%/json_streamer/stream_writer');
         $container->setParameter('.json_streamer.stream_readers_dir', '%kernel.cache_dir%/json_streamer/stream_reader');
         $container->setParameter('.json_streamer.lazy_ghosts_dir', '%kernel.cache_dir%/json_streamer/lazy_ghost');
```

Comment:
This parameter can be removed as well (same in the `src/Symfony/Bundle/FrameworkBundle/Resources/config/json_streamer.php` file)